docs/06-decisions/README.md

```diff
@@ -1,4 +1,4 @@
-# Decisions
+# Record and View Decisions for Parsley
```

Comment:
Do you have a sense of how much space this takes up in the left navbar in docusaurus?

Reply:
Could this be "View Decisions"? I think "for Parsley" is unnecessary, and "record" is maybe not quite right, since docusaurus isn't where people would write this.